src/test/java/uk/gov/hmcts/reform/sscs/service/scheduler/CcdActionExecutorTest.java

```diff
@@ -44,7 +44,7 @@ public void setup() {
 
         caseDetails = SscsCaseDetails.builder().caseTypeId("123").build();
 
-        newSscsCaseData = SscsCaseData.builder().build();
+        SscsCaseData newSscsCaseData = SscsCaseData.builder().build();
```

Comment:
Do you need the "SscsCaseData"?
Looks like it's already up there on line 33.

Reply:
fixed. Thanks,